veniq/baselines/semi/filter.py

```diff
@@ -0,0 +1,109 @@
+from typing import List, Optional, Union
+
+from veniq.ast_framework import AST, ASTNode
+from veniq.ast_framework.block_statement_graph import Block, Statement, build_block_statement_graph
+
+
+ExtractionOpportunity = List[ASTNode]
```

Comment:
Could you clarify why you decided to use this one. Also, after that, as a typing, you use only "List[ASTNode]", for example at the line 32. What is the point?

Reply:
I changed `List[ASTNode]` to `ExtractionOpportunity`